README.md

```diff
@@ -1,5 +1,151 @@
+# The Bowling Challenge
 
-Bowling Challenge
+This weekend (and next) the focus is on JavaScript. This is a totally new language and concept for me. My weekend challenge will be focusing on what have I learned about JavaScript this week, and how can I continue to use the same process to learn more about it. The challenge itself this week is complex so I am going to focus on understanding the business logic first, implementing the user interface second.
```

Comment:
very impressive considering this is totally new language! seemed like someone experienced in it already